### PR TITLE
renaming: err -> errCh

### PR DESCRIPTION
**Pull request by ateleshev**
Minor code refactoring, variable ```err``` was renamed to ```errCh```, see comment: https://github.com/flatcar-linux/flatcar-linux-update-operator/pull/157#pullrequestreview-989184603

### CODE REVIEW

**Review by invidian — DISMISSED** [review]
No need for separate PR, just separate commit would be fine :) LGTM, will merge after CI.

File: pkg/operator/operator.go [hunk 1]
```diff
@@ -197,12 +197,12 @@ func (k *Kontroller) Run(stop <-chan struct{}) error {
...
 
-	return <-err
+	return <-errCh
 }
 
 // withLeaderElection creates a new context which is cancelled when this
 // operator does not hold a lock to operate on the cluster.
-func (k *Kontroller) withLeaderElection(stop <-chan struct{}, err chan<- error) context.Context {
+func (k *Kontroller) withLeaderElection(stop <-chan struct{}, errCh chan<- error) context.Context {
 	leaderElectionBroadcaster := record.NewBroadcaster()
```
Comment (invidian) [inline pkg/operator/operator.go:206]: Oops, linter complains as this code has existing issue and we only report new ones for the time being. Maybe we can take the opportunity and move resource lock creation code to separate function? Like:
```go
func (k *Kontroller) newResourceLock() { ... }
```

Comment (invidian) [reply pkg/operator/operator.go:209]: Alternatively, just add `//nolint:funlen // To be refactored.` comment at the top of this function and I will refactor it myself later.

File: pkg/operator/operator.go [hunk 2]
```diff
@@ -222,12 +222,19 @@ func (k *Kontroller) withLeaderElection(stop <-chan struct{}, err chan<- error)
 		},
 	}
 
+	return resLock
```
Comment (invidian) [inline pkg/operator/operator.go:225]: We can now do simply `return &resourcelock.ConfigMapLock{`, there is no need for intermediate variable.

File: pkg/operator/operator.go (hunk 1 above)
Comment (ateleshev) [reply pkg/operator/operator.go:206]: @invidian done

File: pkg/operator/operator.go (hunk 2 above)
Comment (ateleshev) [reply pkg/operator/operator.go]: @invidian done

**Review by invidian — APPROVED** [review]

**Pull request merged**